packages/semantic-ui-theme/semantic/src/themes/sharegate/collections/form.overrides

```diff
@@ -99,18 +99,19 @@
 
 /* LAYOUT | Sizes */
 .ui.textarea {
-    padding: 1rem;
+    padding: 1rem 1.25rem;
```

Comment:
Should you use our CSS instead?

I am getting worried that if we update a foundation CSS variable value one day, all our apps will be updated but Orbit-UI toolkit will not since we don't use the CSS vars.

Reply:
Yes indeed